src/components/shared/Entity/hooks/useDataFlowResetPrompt.tsx

```diff
@@ -0,0 +1,45 @@
+import { useConfirmationModalContext } from 'context/Confirmation';
+import { useCallback } from 'react';
+import { useBindingStore } from 'stores/Binding/Store';
+import { useShallow } from 'zustand/react/shallow';
+import BindingReview from '../DataflowResetModal/BindingReview';
+
+function useDataFlowResetPrompt() {
+    const confirmationModalContext = useConfirmationModalContext();
```

Comment:
Not 100% sure this is the direction that will be taken. Hoping it will because it has all the things wired up to handle saying "yes" or "no". However, since we need to have multiple "states" in the modal not sure if this will work.